webpack.build.config.js

```diff
@@ -37,15 +36,16 @@ module.exports = {
       }
     ]
   },
+  optimization: {
+    minimize: true,
+    minimizer: [new TerserPlugin()],
```

Comment:
```suggestion
    minimizer: [new TerserPlugin()]
```

Not sure how strict NPM is with syntax, but given how much fun it is otherwise, it's better to be safe than sorry.